packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -194,19 +204,25 @@ Autocomplete.propTypes = {
    */
   loading: PropTypes.bool,
   /**
-   * Message users will see when the `loading` prop is passed to `<Autcomplete />`.
+   * Message users will see when the `loading` prop is passed to `Autcomplete`.
    */
   loadingMessage: PropTypes.node,
   /**
    * Message users will see when the `items` array returns empty and the `loading` prop is passed to `<Autocomplete />`.
    */
   noResultsMessage: PropTypes.node,
   /**
-   * Called when the user selects an item and the selected item has changed. Called with the item that was selected and the new state of `downshift`.
+   * Called when the user selects an item and the selected item has changed. Called with the item that was selected and the new state of `Downshift`.
```

Comment:
We don't really describe to the user what `Downshift` is, so this last sentence is likely confusing. Can we rephrase to:

> Called with the item that was selected and the new state.